From 79d1a8d4f9c89d3513836aba0758e0d2a2a1cfaf Mon Sep 17 00:00:00 2001
From: David Stansby <dstansby@gmail.com>
Date: Mon, 19 Aug 2024 23:04:56 +0100
Subject: [PATCH] fix c compiler setup (#562)

Upstream-Status: Backport [https://github.com/zarr-developers/numcodecs/pull/562]
---
 setup.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/setup.py b/setup.py
index e594a20..815224a 100644
--- a/setup.py
+++ b/setup.py
@@ -8,6 +8,7 @@ from setuptools import Extension, setup
 from setuptools.errors import CCompilerError, ExecError, PlatformError
 from distutils import ccompiler
 from distutils.command.clean import clean
+from distutils.sysconfig import customize_compiler
 
 # determine CPU support for SSE2 and AVX2
 cpu_info = cpuinfo.get_cpu_info()
@@ -325,6 +326,7 @@ class ve_build_ext(build_ext):
             if cpuinfo.platform.machine() == 'x86_64':
                 S_files = glob('c-blosc/internal-complibs/zstd*/decompress/*amd64.S')
                 compiler = ccompiler.new_compiler()
+                customize_compiler(compiler)
                 compiler.src_extensions.append('.S')
                 compiler.compile(S_files)
 
-- 
2.46.0

