--- PyWavelets-1.4.1/pyproject.toml.old	2023-03-12 19:34:07.150757013 +0100
+++ PyWavelets-1.4.1/pyproject.toml	2023-03-12 19:34:32.521011691 +0100
@@ -7,7 +7,7 @@
 [build-system]
 requires = [
     "wheel",
-    "setuptools<65",
+    "setuptools>65",
     "Cython>=0.29.24,<3.0",
 
     # NumPy dependencies - to update these, sync from
@@ -36,7 +36,7 @@
     # however macOS was broken and it's safe C API/ABI-wise to build against 1.21.6
     # (see oldest-supported-numpy issues gh-28 and gh-45)
     "numpy==1.21.6; python_version=='3.10' and (platform_system!='Windows' and platform_machine!='loongarch64') and platform_python_implementation != 'PyPy'",
-    "numpy==1.23.3; python_version=='3.11' and platform_python_implementation != 'PyPy'",
+    "numpy>=1.23.3; python_version=='3.11' and platform_python_implementation != 'PyPy'",
 
     # For Python versions which aren't yet officially supported,
     # we specify an unpinned NumPy which allows source distributions
