From 7d2d046967f2aefc496e6e2517eaa8bc25286b46 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zolt=C3=A1n=20B=C3=B6sz=C3=B6rm=C3=A9nyi?=
 <zboszor@gmail.com>
Date: Tue, 18 Feb 2025 08:56:39 +0100
Subject: [PATCH] Fix referencing Protobuf from the host
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Max <spblizard@gmail.com>
Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [cross-compiler specific]
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7720ed9..89b80fb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -182,11 +182,11 @@
     else()
       find_library(Protobuf_LIBRARY
         NAMES protobuf
-        PATHS ${_PROTOBUF_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
+        PATHS $ENV{STAGING_LIBDIR}
         NO_DEFAULT_PATH)
     endif(ONNX_USE_LITE_PROTO)
     find_path(Protobuf_INCLUDE_DIR google/protobuf/service.h
-              PATHS ${_PROTOBUF_INSTALL_PREFIX}/include
+              PATHS $ENV{STAGING_INCDIR}
               NO_DEFAULT_PATH)
     if(ONNX_USE_PROTOBUF_SHARED_LIBS)
       set(Protobuf_USE_STATIC_LIBS OFF)
-- 
2.48.1

