Upstream-Status: Inappropriate [Yocto specific]
Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>

diff -durpN imagecodecs-2025.3.30.old/setup.py imagecodecs-2025.3.30/setup.py
--- imagecodecs-2025.3.30.old/setup.py	2025-03-22 05:53:53.000000000 +0100
+++ imagecodecs-2025.3.30/setup.py	2025-06-02 16:21:27.399072257 +0200
@@ -263,14 +263,16 @@ def customize_build_default(EXTENSIONS,
     Works on Ubuntu 24.04
 
     """
+    INCLIB = os.environ.get('INCLIB', '.')
+
     EXTENSIONS['jpeg2k']['include_dirs'].extend(
         (
-            '/usr/include/openjpeg-2.3',
-            '/usr/include/openjpeg-2.4',
-            '/usr/include/openjpeg-2.5',
+            os.path.join(INCLIB, 'include', 'openjpeg-2.3'),
+            os.path.join(INCLIB, 'include', 'openjpeg-2.4'),
+            os.path.join(INCLIB, 'include', 'openjpeg-2.5'),
         )
     )
-    EXTENSIONS['jpegxr']['include_dirs'].append('/usr/include/jxrlib')
+    # EXTENSIONS['jpegxr']['include_dirs'].append('/usr/include/jxrlib')
 
     if not os.environ.get('IMAGECODECS_JPEG8_LEGACY', False):
         # use libjpeg-turbo 3 by default
@@ -280,17 +282,25 @@ def customize_build_default(EXTENSIONS,
     keep = {
         'shared',
         'imcd',
+        'aec',
+        'avif',
+        'blosc',
+        'blosc2',
         'bmp',
+        'brotli',
+        'bz2',
         'cms',
         'deflate',
+        'heif',
         'jpeg2k',
         'jpeg8',
         # 'jpegxl',  # requires v0.10
-        'jpegxr',
-        'lerc',  # requires v4
+        # 'jpegxr',
+        # 'lerc',  # requires v4
         'lz4',
         'lzma',
         'png',
+        'snappy',
         'tiff',  # requires v4.6
         'webp',
         'zlib',
