Upstream-Status: Inappropriate [Yocto specific]
Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>

diff -durpN imagecodecs-2026.3.6.old/setup.py imagecodecs-2026.3.6/setup.py
--- imagecodecs-2026.3.6.old/setup.py	2026-03-04 00:23:22.000000000 +0100
+++ imagecodecs-2026.3.6/setup.py	2026-03-24 06:19:20.513553601 +0100
@@ -294,16 +294,18 @@ def customize_build_default(
     Works on Ubuntu 24.04
 
     """
+    INCLIB = os.environ.get('INCLIB', '.')
+
     del options['shared_utility_qualified_name']
 
     extensions['jpeg2k']['include_dirs'].extend(
         (
-            '/usr/include/openjpeg-2.3',
-            '/usr/include/openjpeg-2.4',
-            '/usr/include/openjpeg-2.5',
+            os.path.join(INCLIB, 'include', 'openjpeg-2.3'),
+            os.path.join(INCLIB, 'include', 'openjpeg-2.4'),
+            os.path.join(INCLIB, 'include', 'openjpeg-2.5'),
         )
     )
-    extensions['jpegxr']['include_dirs'].append('/usr/include/jxrlib')
+    # extensions['jpegxr']['include_dirs'].append('/usr/include/jxrlib')
 
     if not os.environ.get('IMAGECODECS_JPEG8_LEGACY', ''):
         # use libjpeg-turbo 3 by default
@@ -313,17 +315,25 @@ def customize_build_default(
     keep = {
         'shared',
         'imcd',
+        'aec',
+        'avif',
+        'blosc',
+        'blosc2',
         'bmp',
+        'brotli',
+        'bz2',
         'cms',
         'deflate',
+        'heif',
         'jpeg2k',
         'jpeg8',
         # 'jpegxl',  # requires v0.10
-        'jpegxr',
-        'lerc',  # requires v4
+        # 'jpegxr',
+        # 'lerc',  # requires v4
         'lz4',
         'lzma',
         'png',
+        'snappy',
         'tiff',  # requires v4.6
         'webp',
         'zlib',
