Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [oe specific]

diff -durpN numba-0.61.2.old/PKG-INFO numba-0.61.2/PKG-INFO
--- numba-0.61.2.old/PKG-INFO	2025-04-07 18:34:47.977649000 +0200
+++ numba-0.61.2/PKG-INFO	2025-09-18 15:44:19.021107861 +0200
@@ -19,7 +19,7 @@ Requires-Python: >=3.10
 License-File: LICENSE
 License-File: LICENSES.third-party
 Requires-Dist: llvmlite<0.45,>=0.44.0dev0
-Requires-Dist: numpy<2.3,>=1.24
+Requires-Dist: numpy<2.4,>=1.24
 
 *****
 Numba
diff -durpN numba-0.61.2.old/setup.py numba-0.61.2/setup.py
--- numba-0.61.2.old/setup.py	2025-04-07 18:34:14.000000000 +0200
+++ numba-0.61.2/setup.py	2025-09-18 15:44:24.735105012 +0200
@@ -23,7 +23,7 @@ min_python_version = "3.10"
 max_python_version = "3.14"  # exclusive
 min_numpy_build_version = "2.0.0rc1"
 min_numpy_run_version = "1.24"
-max_numpy_run_version = "2.3"
+max_numpy_run_version = "2.4"
 min_llvmlite_version = "0.44.0dev0"
 max_llvmlite_version = "0.45"
 
