From 6c4eb6a206a63817c2c365df4bffc787c9ceb7b7 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Fri, 2 Jan 2026 20:02:57 +0100
Subject: [PATCH] relax cython version maximum

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Inappropriate [oe-specific]
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 01171ca..e613cc8 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,7 @@ requires = [
     # We need at least Cython 3.1.0 for free-threaded CPython;
     # for other CPython versions, the regular pre-emptive
     # Cython version bounds policy applies
-    "Cython>=3.0.8,<3.2.0",
+    "Cython>=3.0.8,<3.2.4",
     # The upper bound on pybind11 is pre-emptive only
     "pybind11>=2.13.2,<3.1.0",
     # The upper bound on pythran is pre-emptive only; 0.18.0
