diff --git a/scipy/stats/setup.py b/scipy/stats/setup.py
index 52a5a36f0..fb5c16a6b 100644
--- a/scipy/stats/setup.py
+++ b/scipy/stats/setup.py
@@ -64,10 +64,10 @@ def configuration(parent_package='', top_path=None):
         biasedurn_libs = []
         biasedurn_libdirs = []
     else:
-        biasedurn_libs = ['npyrandom', 'npymath']
-        biasedurn_libdirs = [join(np.get_include(),
-                                  '..', '..', 'random', 'lib')]
-        biasedurn_libdirs += get_info('npymath')['library_dirs']
+        biasedurn_libs = [join(os.environ["NUMPY_INCLUDE_PATH"],
+                                  '..', '..', 'random', 'lib', 'libnpyrandom.a')]
+        biasedurn_libs.append([join(os.environ["NUMPY_INCLUDE_PATH"],
+                                  '..', '..', 'core', 'lib', 'libnpymath.a')])
 
     ext = config.add_extension(
         '_biasedurn',
@@ -79,11 +79,10 @@ def configuration(parent_package='', top_path=None):
             'biasedurn/stoc1.cpp',
             'biasedurn/stoc3.cpp'],
         include_dirs=[np.get_include()],
-        library_dirs=biasedurn_libdirs,
-        libraries=biasedurn_libs,
         define_macros=[('R_BUILD', None)],
         language='c++',
         depends=['biasedurn/stocR.h'],
+        extra_objects=biasedurn_libs,
     )
     ext._pre_build_hook = pre_build_hook
 
