Upstream-Status: Inappropriate [Yocto specific]
Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>

diff -durpN scipy-1.13.0.old/pyproject.toml scipy-1.13.0/pyproject.toml
--- scipy-1.13.0.old/pyproject.toml	2024-04-02 20:31:54.000000000 +0200
+++ scipy-1.13.0/pyproject.toml	2024-04-12 09:13:04.910776142 +0200
@@ -35,7 +35,7 @@ requires = [
     #      and disabling build isolation.
     #   3. The <2.3 upper bound is for matching the numpy deprecation policy,
     #      it should not be loosened.
-    "numpy>=2.0.0rc1,<2.3",
+    "numpy",
 ]
 
 [project]
