Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]

diff -durpN pythran-0.16.1.old/PKG-INFO pythran-0.16.1/PKG-INFO
--- pythran-0.16.1.old/PKG-INFO	2024-05-28 07:18:01.924732700 +0200
+++ pythran-0.16.1/PKG-INFO	2024-10-15 14:22:33.386386069 +0200
@@ -54,7 +54,7 @@ License-File: LICENSE
 License-File: AUTHORS
 Requires-Dist: ply>=3.4
 Requires-Dist: setuptools
-Requires-Dist: gast~=0.5.0
+Requires-Dist: gast>=0.5.0
 Requires-Dist: numpy
 Requires-Dist: beniget~=0.4.0
 Provides-Extra: doc
diff -durpN pythran-0.16.1.old/requirements.txt pythran-0.16.1/requirements.txt
--- pythran-0.16.1.old/requirements.txt	2023-12-11 09:19:50.000000000 +0100
+++ pythran-0.16.1/requirements.txt	2024-10-15 14:22:33.386386069 +0200
@@ -1,5 +1,5 @@
 ply>=3.4
 setuptools
-gast~=0.5.0
+gast>=0.5.0
 numpy
 beniget~=0.4.0
