Upstream-Status: Inappropriate [Yocto specific]
Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>

diff -durpN blosc2-4.14.0.orig/CMakeLists.txt blosc2-4.14.0/CMakeLists.txt
--- blosc2-4.14.0.orig/CMakeLists.txt	2022-11-09 13:37:21.000000000 +0100
+++ blosc2-4.14.0/CMakeLists.txt	2025-05-05 09:36:38.823950090 +0200
@@ -64,6 +64,7 @@
   OUTPUT blosc2_ext.c
   COMMAND Python::Interpreter -m cython
           "${CMAKE_CURRENT_SOURCE_DIR}/src/blosc2/blosc2_ext.pyx" --output-file blosc2_ext.c
+  COMMAND sed -i "s:../../..${STAGING_DIR_NATIVE}::g" blosc2_ext.c
   DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/src/blosc2/blosc2_ext.pyx"
   VERBATIM)
 
@@ -85,6 +86,7 @@
   OUTPUT utf8_ext.c
   COMMAND Python::Interpreter -m cython
           "${CMAKE_CURRENT_SOURCE_DIR}/src/blosc2/utf8_ext.pyx" --output-file utf8_ext.c
+  COMMAND sed -i "s:../../..${STAGING_DIR_NATIVE}::g" utf8_ext.c
   DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/src/blosc2/utf8_ext.pyx"
   VERBATIM)
 
