From 7cef04fdf7d65df88ee6ce695d0bd50614e4432b Mon Sep 17 00:00:00 2001
From: Buildbot <buildbot@xenial.com>
Date: Thu, 1 Sep 2022 12:23:22 +0000
Subject: [PATCH] Pass through EXTRA_OECMAKE
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]
---
 tools/setup_helpers/cmake.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tools/setup_helpers/cmake.py b/tools/setup_helpers/cmake.py
index 21943619329..44812e18eca 100644
--- a/tools/setup_helpers/cmake.py
+++ b/tools/setup_helpers/cmake.py
@@ -414,6 +414,11 @@ class CMake:
                         file=sys.stderr,
                     )
                     print(e, file=sys.stderr)
+
+        # Add Yocto specific configuration
+        for arg in os.getenv("EXTRA_OECMAKE", "").split():
+            args.append(arg)
+
         # According to the CMake manual, we should pass the arguments first,
         # and put the directory as the last element. Otherwise, these flags
         # may not be passed correctly.
-- 
2.31.1

