From 0bd1ab3d8be2f6926bde2f3363210da8560a66cc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zolt=C3=A1n=20B=C3=B6sz=C3=B6rm=C3=A9nyi?=
 <zboszor@gmail.com>
Date: Mon, 4 Aug 2025 11:41:38 +0200
Subject: [PATCH] Fix gloo with GCC 15
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]
---
 gloo/types.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gloo/types.h b/gloo/types.h
index cd38c8a..49df1ad 100644
--- a/gloo/types.h
+++ b/gloo/types.h
@@ -4,6 +4,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <iostream>
 
 #ifdef __CUDA_ARCH__
-- 
2.50.1

