Upstream-Status: Inappropriate [Yocto specific]
Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>

--- astropy-6.1.2/pyproject.toml~	2024-07-23 16:15:00.000000000 +0200
+++ astropy-6.1.2/pyproject.toml	2024-08-21 15:46:21.945024095 +0200
@@ -128,7 +128,7 @@
 requires = ["setuptools",
             "setuptools_scm>=6.2",
             "cython>=3.0.0,<3.1.0",
-            "numpy>=2.0.0",
+            "numpy",
             "extension-helpers==1.*"]
 build-backend = "setuptools.build_meta"
 
