--- astropy-5.2.1/pyproject.toml.old	2023-03-12 19:10:56.067305683 +0100
+++ astropy-5.2.1/pyproject.toml	2023-03-27 13:05:18.932740834 +0200
@@ -2,8 +2,8 @@
 requires = ["setuptools",
             "setuptools_scm>=6.2",
             "wheel",
-            "cython==0.29.30",
-            "oldest-supported-numpy",
+            "numpy",
+            "cython>=0.29.30",
             "extension-helpers"]
 build-backend = 'setuptools.build_meta'
 
