From bc259763b5bf5bee55d6408a68bed6467c0fd68f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zolt=C3=A1n=20B=C3=B6sz=C3=B6rm=C3=A9nyi?=
 <zboszor@gmail.com>
Date: Wed, 7 Jun 2023 14:32:30 +0200
Subject: [PATCH 2/2] Use unversioned numpy dependency
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [OE specific]
---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertion(+), 2 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index ebfb70f7b..885bb3963 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -53,7 +53,7 @@
     'ninja',
     'Cython>=3.0.4',
     'pythran',
-    'numpy>=2.0.0rc1',
+    'numpy',
     # Developer UI
     'spin==0.8',
     'build',
@@ -126,7 +126,7 @@
     'Cython>=3.0.4',
     'pythran',
     'lazy_loader>=0.3',
-    'numpy>=2.0.0rc1',
+    'numpy',
 ]
 
 [tool.spin]
-- 
2.40.1

