Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]

--- numba-0.60.0rc1/setup.py.old	2024-05-23 05:28:22.180837983 +0200
+++ numba-0.60.0rc1/setup.py	2024-05-23 05:28:41.155055481 +0200
@@ -21,7 +21,7 @@ except ImportError:
 
 min_python_version = "3.9"
 max_python_version = "3.13"  # exclusive
-min_numpy_build_version = "2.0.0rc1"
+min_numpy_build_version = "1.22"
 min_numpy_run_version = "1.22"
 max_numpy_run_version = "2.1"
 min_llvmlite_version = "0.43.0dev0"
