From 54e1df791ce5ba2bbacd43cf11225ba2e06fd58b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zolt=C3=A1n=20B=C3=B6sz=C3=B6rm=C3=A9nyi?=
 <zboszor@gmail.com>
Date: Wed, 14 May 2025 16:04:21 +0200
Subject: [PATCH 2/2] Pass presence and version of ffmpeg via envvar
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]
---
 setup.py | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/setup.py b/setup.py
index 15df41a528..ec016dff71 100644
--- a/setup.py
+++ b/setup.py
@@ -375,20 +375,15 @@ def make_video_decoders_extensions():
         print("Can only build video decoder extensions on linux and Python != 3.9")
         return []
 
-    ffmpeg_exe = shutil.which("ffmpeg")
-    if ffmpeg_exe is None:
-        print(f"{build_without_extensions_msg} Couldn't find ffmpeg binary.")
-        return []
-
     def find_ffmpeg_libraries():
         ffmpeg_libraries = {"libavcodec", "libavformat", "libavutil", "libswresample", "libswscale"}
 
-        ffmpeg_bin = os.path.dirname(ffmpeg_exe)
-        ffmpeg_root = os.path.dirname(ffmpeg_bin)
+        ffmpeg_root = os.getenv("FFMPEG_ROOT")
         ffmpeg_include_dir = os.path.join(ffmpeg_root, "include")
         ffmpeg_library_dir = os.path.join(ffmpeg_root, "lib")
 
-        gcc = os.environ.get("CC", shutil.which("gcc"))
+        gcccmd = os.environ.get("CC", shutil.which("gcc"))
+        gcc = gcccmd.split()[0]
         platform_tag = subprocess.run([gcc, "-print-multiarch"], stdout=subprocess.PIPE)
         platform_tag = platform_tag.stdout.strip().decode("utf-8")
 
-- 
2.49.0

