Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]

diff -durpN scikit_build-0.19.0.old/skbuild/cmaker.py scikit_build-0.19.0/skbuild/cmaker.py
--- scikit_build-0.19.0.old/skbuild/cmaker.py	2026-03-23 15:59:30.746570044 +0100
+++ scikit_build-0.19.0/skbuild/cmaker.py	2026-03-23 15:57:45.576838158 +0100
@@ -247,7 +247,7 @@ class CMaker:
             with contextlib.suppress(ImportError):
                 import ninja  # noqa: PLC0415
 
-                ninja_executable_path = os.path.join(ninja.BIN_DIR, "ninja")
+                ninja_executable_path = "ninja"
 
         if not os.path.exists(CMAKE_BUILD_DIR()):
             os.makedirs(CMAKE_BUILD_DIR())
diff -durpN scikit_build-0.19.0.old/skbuild/platform_specifics/unix.py scikit_build-0.19.0/skbuild/platform_specifics/unix.py
--- scikit_build-0.19.0.old/skbuild/platform_specifics/unix.py	2026-03-23 15:59:30.746785207 +0100
+++ scikit_build-0.19.0/skbuild/platform_specifics/unix.py	2026-03-23 15:57:45.586838145 +0100
@@ -17,7 +17,7 @@ class UnixPlatform(abstract.CMakePlatfor
         try:
             import ninja  # noqa: PLC0415
 
-            ninja_executable_path = os.path.join(ninja.BIN_DIR, "ninja")
+            ninja_executable_path = "ninja"
             ninja_args = [f"-DCMAKE_MAKE_PROGRAM:FILEPATH={ninja_executable_path}"]
         except ImportError:
             ninja_args = []
