From e95e31a9e93b96c6313bf4663c0c0c4c69e49170 Mon Sep 17 00:00:00 2001
From: Mike Taves <mwtoews@gmail.com>
Date: Tue, 20 Aug 2024 10:14:01 +1200
Subject: [PATCH 1/2] MAINT: add Python 3.13 Trove classifier

Upstream-Status: Backport [e95e31a9e93b96c6313bf4663c0c0c4c69e49170]
---
 pyproject.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyproject.toml b/pyproject.toml
index 71b554a..cbba3cb 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -39,6 +39,7 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Topic :: Scientific/Engineering :: GIS",
 ]
 requires-python = ">=3.7"
-- 
2.47.0

