From a2d37eb65d01d76e5da1328ef80c421c4f5a39fd Mon Sep 17 00:00:00 2001
From: Ben Beasley <code@musicinmybrain.net>
Date: Mon, 4 Dec 2023 14:36:36 -0500
Subject: [PATCH 2/2] Fix incompatible pointer type passed to GEOSPolygonize_r
 (#1945)

Upstream-Status: Backport [a2d37eb65d01d76e5da1328ef80c421c4f5a39fd]
---
 src/ufuncs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ufuncs.c b/src/ufuncs.c
index dbd170a..a656955 100644
--- a/src/ufuncs.c
+++ b/src/ufuncs.c
@@ -2178,7 +2178,7 @@ static void polygonize_func(char** args, const npy_intp* dimensions, const npy_i
 
   GEOS_INIT;
 
-  GEOSGeometry** geoms = malloc(sizeof(void*) * dimensions[1]);
+  const GEOSGeometry** geoms = malloc(sizeof(void*) * dimensions[1]);
   if (geoms == NULL) {
     errstate = PGERR_NO_MALLOC;
     goto finish;
-- 
2.47.0

